remotion/compositions/showcases/very-tech-trip/TalkTitle.tsx

```diff
@@ -0,0 +1,44 @@
+import {interpolate, spring, useCurrentFrame, useVideoConfig} from 'remotion';
+import {Title} from '../../../design/atoms/Title';
+
+export const TalkTitle: React.FC<{
+	title: string;
+	delay?: number;
+}> = ({title}) => {
+	const frame = useCurrentFrame();
+	const {fps} = useVideoConfig();
+
+	const titleOpacity = spring({
+		frame,
+		fps,
+		from: 0,
+		to: 1,
+		durationInFrames: 60,
+	});
+
+	const titleDeblur = interpolate(frame, [0, 20], [5, 0], {
+		extrapolateRight: 'clamp',
+	});
+
+	return (
+		<Title
+			style={
+				{
+					'-webkit-line-clamp': '3',
+					bottom: '130px',
+					color: 'white',
+					filter: `blur(${titleDeblur}px)`,
+					fontSize: '32px',
+					opacity: titleOpacity,
+					padding: '0 25px',
+					position: 'absolute',
+					textAlign: 'center',
+					textShadow: 'px 2px 2px #e95900',
+					width: '100%',
+				} as React.CSSProperties
```

Comment:
Des fois tu types des fois non ?

Reply:
en fait je type quand y a une propriété comme '-webkit-line-clamp' sinon il met une erreur de type.

Reply:
Je ne sais pourquoi mais j'ai reussi a les passer en css in js alors qu'avant non, peut etre un fix de react jsp mais j'ai réussi a enlever le typage en plus du coup